### PR TITLE
Some minor improvements and cosmetic changes on UI

### PR DESCRIPTION
**Pull request by odil24**

### CODE REVIEW

**Comment by neo2068** [comment]
I tested the pull request and I got a few errors or problems, e.g. the icon keys for 'link' and 'intel' are missing.
I would add the commits one by one. I will start with the bug fixes. I do not agree with some changes in the UI. We have to discuss how to deal with that.

**Comment by odil24** [comment]
> I tested the pull request and I got a few errors or problems, e.g. the icon keys for 'link' and 'intel' are missing. I would add the commits one by one. I will start with the bug fixes. I do not agree with some changes in the UI. We have to discuss how to deal with that.

Hi. I fixed bug with missed files and you can recheck pull. So what are mistaken from my side wich you don't agree?

File: ui/render/denoiser.py [hunk 1]
```diff
@@ -38,22 +39,23 @@ def draw(self, context):
...
-            sub.prop(denoiser, "hist_dist_thresh")
-            sub = layout.column(align=True)
-            sub.prop(denoiser, "search_window_radius")
-        elif denoiser.type == "OIDN":
+            #sub.prop(denoiser, "filter_spikes")
+            #sub = layout.column(align=True)
+            #sub.prop(denoiser, "hist_dist_thresh")
+            #sub = layout.column(align=True)
+            #sub.prop(denoiser, "search_window_radius")
+        #elif denoiser.type == "OIDN":
```
Comment (BeheadedKamikaze) [inline ui/render/denoiser.py:57]: There's a _lot_ of commented-out code in this PR. If you're intending to remove it, delete the lines. The original lines will still be available in the commit history.

**Review by BeheadedKamikaze — CHANGES_REQUESTED** [review]

**Comment by BeheadedKamikaze** [comment]
I'd also recommend to rebase the branch on `master` now that 6318b6c is merged.

**Comment by odil24** [comment]
> I'd also recommend to rebase the branch on `master` now that [6318b6c](https://github.com/LuxCoreRender/BlendLuxCore/commit/6318b6caa906946eda0852dcf420b88f52810d97) is merged.

I don't understand by this.

**Comment by neo2068** [comment]
> Hi. I fixed bug with missed files and you can recheck pull. So what are mistaken from my side wich you don't agree?
I don't agree with removing some features from the UI, e.g. the random sampler and some pixel filter kernels.

**Comment by odil24** [comment]
> 

Ok. I will change.

**Comment by BeheadedKamikaze** [comment]
> > I'd also recommend to rebase the branch on `master` now that [6318b6c](https://github.com/LuxCoreRender/BlendLuxCore/commit/6318b6caa906946eda0852dcf420b88f52810d97) is merged.
> 
> I don't understand by this.

A lot of the changes in this PR have already been merged in the other commit, so they're conflicting. That's why you are seeing this message:

> image

Recommend you look into the [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) feature of Git so you can replay your changes, resolve any conflicts, and maybe even squash some of these (very many) commits together.

File: __init__.py [hunk 2]
```diff
@@ -66,13 +66,13 @@ def get_bin_directory():
...
 
 bl_info = {
-    "name": "LuxCoreRender is a physically based and unbiased rendering engine",
-    "author": "Odilkhan Yakubov (odil24), Simon Wendsche (B.Y.O.B.), Michael Klemm (neo2068), acasta69, u3dreal, Philstix",
-    "version": (2, 61.1),
-    "blender": (3, 0, 0),
+    "name": "LuxCoreRender",
+    "author": "Simon Wendsche (B.Y.O.B.), Michael Klemm (neo2068), Odilkhan Yakubov (odil24), acasta69, u3dreal, Philstix",
+    "version": (2, 61),
+    "blender": (3, 4, 0),
```
Comment (BeheadedKamikaze) [inline __init__.py:72]: Are you removing support for Blender 3.0, 3.1, 3.2, and 3.3?

Comment (odil24) [reply __init__.py:72]: Thats because LuxCore (latest) works flawless from Blender 3.4 than previous versions which is not supported new "index_attribute" operator which fixed by @neo2068 for material indexing (Multi-material fixed):
https://github.com/LuxCoreRender/BlendLuxCore/commit/6318b6caa906946eda0852dcf420b88f52810d97

Comment (odil24) [reply __init__.py:72]: > Are you removing support for Blender ~3.0~, 3.1, 3.2, and 3.3?
> 
> We usually like to retain support at least for the LTS version (3.3), and even older versions if there is no technical reason why they should be incompatible.

Sorry, but I don't understand what here a link is written because I'm not familiar with that.

**Comment by odil24** [comment]
@neo2068 @BeheadedKamikaze please, check. I think all is fine now.

File: __init__.py (hunk 2 above)
Comment (neo2068) [reply __init__.py:72]: > Are you removing support for Blender ~3.0~, 3.1, 3.2, and 3.3?

No, the versions are still supported. I changed the version back to 3.0. I had interpreted the version as the latest supported version but in the documentation it is listed as minimum supported version.

Comment (BeheadedKamikaze) [reply __init__.py:72]: OK that's great, thanks @neo2068

Comment (odil24) [reply __init__.py:72]: @neo2068 hi. Why are you did not include my latest changes related to the icons? Is my idea bad?

**Comment by neo2068** [comment]
@odil24 Hi! I like the idea but there are still missing files (e.g. help icon) and I have to review it completely. We should add the icons also to the other UI panels which are special to LuxcoreRender.

**Comment by odil24** [comment]
> @odil24 Hi! I like the idea but there are still missing files (e.g. help icon) and I have to review it completely. We should add the icons also to the other UI panels which are special to LuxcoreRender.

Hi. As you can see I already fixed missed files and pulled request.

**Comment by neo2068** [comment]
I have seen, that you fixed some missing files but even with this last fix there are still errors on my side.
```
Traceback (most recent call last):
  File "C:\Users\Michael Klemm\AppData\Local\Programs\Blender\blender-3.5.0-alpha+master.8709a51fa907-windows.amd64-release\3.5\scripts\modules\bpy_types.py", line 903, in draw_ls
    func(self, context)
  File "C:\Users\Michael Klemm\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\BlendLuxCore\ui\render\config.py", line 44, in luxcore_render_draw
    row.operator("luxcore.render_settings_helper", icon_value= icon_manager.get_icon_id("help"))
  File "C:\Users\Michael Klemm\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\BlendLuxCore\ui\render\icons.py", line 48, in get_icon_id
    return self.icon_previews[icon_name].icon_id
KeyError: 'help'
```

**Comment by odil24** [comment]
> I have seen, that you fixed some missing files but even with this last fix there are still errors on my side.
> 
> ```
> Traceback (most recent call last):
>   File "C:\Users\Michael Klemm\AppData\Local\Programs\Blender\blender-3.5.0-alpha+master.8709a51fa907-windows.amd64-release\3.5\scripts\modules\bpy_types.py", line 903, in draw_ls
>     func(self, context)
>   File "C:\Users\Michael Klemm\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\BlendLuxCore\ui\render\config.py", line 44, in luxcore_render_draw
>     row.operator("luxcore.render_settings_helper", icon_value= icon_manager.get_icon_id("help"))
>   File "C:\Users\Michael Klemm\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\BlendLuxCore\ui\render\icons.py", line 48, in get_icon_id
>     return self.icon_previews[icon_name].icon_id
> KeyError: 'help'
> ```

Hi. I think my latest commit should fixed an issue. Check again. Also is it not possible BCD denoiser or it is useable option?

**Comment by neo2068** [comment]
I will merge your changes regarding the luxcore icon.

**Comment by neo2068** [comment]
Hi! BCD denoiser is a valid option and should not be removed.

**Comment by odil24** [comment]
@neo2068 please, check the commits.

File: ui/camera.py [hunk 3]
```diff
@@ -74,10 +77,11 @@ class LUXCORE_CAMERA_PT_clipping_plane(CameraButtonsPanel, Panel):
 
     def draw_header(self, context):
         layout = self.layout
+        layout.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
         col = layout.column(align=True)
         col.prop(context.camera.luxcore, "use_clipping_plane", text="")
         col = layout.column(align=True)
-        col.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
+        #col.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
```
Comment (BeheadedKamikaze) [inline ui/camera.py:84]: Duplicate code (also commented out?)

File: properties/denoiser.py [hunk 4]
```diff
@@ -31,8 +31,8 @@ class LuxCoreDenoiser(PropertyGroup):
 
     enabled: BoolProperty(name="", default=False, description="Enable/disable denoiser")
     type_items = [
-        ("BCD", "BCD", "Bayesian Collaborative Denoiser", 0),
-        ("OIDN", "OIDN", "Intel Open Image Denoiser", 1),
+        #("BCD", "BCD", "Bayesian Collaborative Denoiser", 0),
+        ("OIDN", "OpenImageDenoiser", "", 0),
```
Comment (BeheadedKamikaze) [inline properties/denoiser.py:35]: I thought you were not going to remove the BCD denoiser?

File: properties/config.py [hunk 5]
```diff
@@ -425,9 +424,9 @@ class LuxCoreConfig(PropertyGroup):
     sampler: EnumProperty(name="Sampler", items=samplers, default="SOBOL")
     
     samplers_gpu = [
-        ("SOBOL", "Sobol", "Best suited sampler for the GPU. " + SOBOL_DESC, 0),
-        ("RANDOM", "Random", RANDOM_DESC, 1),
-    ]
+        ("SOBOL", "Sobol", "Best suited sampler for the GPU. " + SOBOL_DESC, 0),]
+        #("RANDOM", "Random", RANDOM_DESC, 1),
+    #]
```
Comment (BeheadedKamikaze) [inline properties/config.py:429]: Why was this removed? I think it's sometimes used by the BCD denoiser.

Also why is the code commented out rather than deleted?

File: ui/render/image_resize_policy.py [hunk 6]
```diff
@@ -14,10 +14,11 @@ def poll(cls, context):
 
     def draw_header(self, context):
         layout = self.layout
+        layout.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
         col = layout.column(align=True)
         col.prop(context.scene.luxcore.config.image_resize_policy, "enabled", text="")
         col = layout.column(align=True)
-        col.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
+        #col.label(text="", icon_value=icon_manager.get_icon_id("logotype"))
```
Comment (BeheadedKamikaze) [inline ui/render/image_resize_policy.py:21]: Duplicate code (also commented out?)

File: ui/render/sampling.py [hunk 7]
```diff
@@ -60,7 +67,7 @@ def draw(self, context):
             else:
                 row.prop(config, "sampler")
 
-            if sampler in {"SOBOL", "RANDOM"}:
+            if sampler in "SOBOL":
```
Comment (BeheadedKamikaze) [inline ui/render/sampling.py:70]: Needs to be reverted

File: ui/render/sampling.py [hunk 8]
```diff
@@ -73,8 +80,8 @@ def draw(self, context):
                     if config.using_out_of_core():
                         col.prop(config, "out_of_core_supersampling")
             elif sampler == "METROPOLIS":
-                if denoiser.enabled and denoiser.type == "BCD":
-                    layout.label(text="Can lead to artifacts in the denoiser!", icon=icons.WARNING)
+                #if denoiser.enabled and denoiser.type == "BCD":
+                    #layout.label(text="Can lead to artifacts in the denoiser!", icon=icons.WARNING)
```
Comment (BeheadedKamikaze) [inline ui/render/sampling.py:84]: Why is there so much commented-out code?

Also this goes with the BCD denoiser so also needs to be reverted

File: ui/render/sampling.py [hunk 9]
```diff
@@ -218,7 +221,3 @@ def draw(self, context):
             col.active = False
         
         col.prop(config, "light_strategy")
-        
-        # Tiled path
-        if config.engine == "PATH":
-            layout.prop(config, "use_tiles")
```
Comment (BeheadedKamikaze) [inline ui/render/sampling.py:224]: Why are you removing this?

**Review by BeheadedKamikaze — COMMENTED** [review]
Why is there so much commented-out code?

Also it was suggested that you do not remove the BCD denoiser, the random sampler (used by BCD), and I'm also unsure why you removed the UI for Tiled Rendering (seems completely unrelated and unnecessary to remove).

Can you please address these issues?

**Comment by odil24** [comment]
@BeheadedKamikaze Hi. Yes, you definetly right and I need to revert it again. Thanks.

File: ui/render/sampling.py (hunk 8 above)
Comment (neo2068) [reply ui/render/sampling.py:84]: The pull request is merged and I removed the commented code.

File: ui/render/sampling.py (hunk 9 above)
Comment (neo2068) [reply ui/render/sampling.py:224]: The option is moved to the normal sampling panel. i.e. line 47 ff.

Comment (BeheadedKamikaze) [reply ui/render/sampling.py:224]: Ah you're right, thanks I missed it
